src/lava/magma/core/model/py/connection.py

```diff
@@ -60,7 +57,7 @@ def __init__(self, proc_params: dict) -> None:
         # add all necessary ports get access to all learning params
         self._learning_rule: LoihiLearningRule = proc_params["learning_rule"]
         self._shape: typing.Tuple[int, ...] = proc_params["shape"]
-        self._graded_input: typing.Optional[bool] = proc_params["graded_input"]
+        # TODO move that somewhere else, if that's really needed! self._graded_input: typing.Optional[bool] = proc_params["graded_input"]
```

Comment:
So if I get a graded reward input trace, how would I receive it and adjust the third-factor trace accordingly?